src/snyk/common/configuration/configuration.ts

```diff
@@ -287,7 +321,9 @@ export class Configuration implements IConfiguration {
   }
 
   async setCliPath(cliPath: string | undefined): Promise<void> {
-    if (!cliPath) return;
+    if (!cliPath && extensionContext) {
+      cliPath = await CliExecutable.getPath(extensionContext.extensionPath);
```

Comment:
why do we set it to extensionPath? shouldn't it be a subfolder? Or include the filename?

Reply:
```
  static async getPath(extensionDir: string, customPath?: string): Promise<string> {
    if (customPath) {
      return customPath;
    }

    const platform = await this.getCurrentWithArch();
    const fileName = this.getFileName(platform);
    return path.join(extensionDir, fileName);
  }
```

Reply:
the function name is confusing 
maybe it should be determineCliPath or smth